tests/sentry/grouping/ingest/test_seer.py

```diff
@@ -306,3 +308,47 @@ def test_returns_no_grouphash_and_empty_metadata_if_no_similar_group_found(self)
                 expected_metadata,
                 None,
             )
+
+    @patch("sentry.seer.similarity.utils.logger")
+    def test_too_many_only_system_frames(self, mock_logger: Mock) -> None:
+        type = "FailedToFetchError"
+        value = "Charlie didn't bring the ball back"
+        context_line = f"raise {type}('{value}')"
+        new_event = Event(
+            project_id=self.project.id,
+            event_id="22312012112120120908201304152013",
+            data={
+                "title": f"{type}('{value}')",
+                "exception": {
+                    "values": [
+                        {
+                            "type": type,
+                            "value": value,
+                            "stacktrace": {
+                                "frames": [
+                                    {
+                                        "function": f"play_fetch_{i}",
+                                        "filename": f"dogpark{i}.py",
+                                        "context_line": context_line,
+                                    }
+                                    for i in range(MAX_FRAME_COUNT + 1)
+                                ]
+                            },
+                        }
+                    ]
+                },
+                "platform": "python",
+            },
+        )
+        variants = new_event.get_grouping_variants()
+        get_seer_similar_issues(new_event, variants)
+
+        grouping_info = get_grouping_info_from_variants(variants)
+        mock_logger.info.assert_called_with(
+            "grouping.similarity.over_threshold_system_only_frames",
+            extra={
+                "project_id": "",
+                "event_id": "",
+                "hash": grouping_info["system"]["hash"],
+            },
+        )
```

Comment:
Nit:

```suggestion
        grouping_info = get_grouping_info_from_variants(variants)
        grouping_info["event_id"] = 11212012123120120415201309082013
        grouping_info["project_id"] = 12311121
        mock_logger.info.assert_called_with(
            "grouping.similarity.over_threshold_system_only_frames",
            extra={
                "project_id": 11212012123120120415201309082013,
                "event_id": 12311121,
                "hash": grouping_info["system"]["hash"],
            },
        )
```

Reply:
the case we're testing here is calling `get_seer_similar_issues` without modifying `grouping_info` first. so the case when this [line](https://github.com/getsentry/sentry/pull/81130/files#diff-ab297a1c8837d7ad14a3df1d89d9ba64462cb7193c0d001bec510f60a14706e0L219-L221) is called.

this also doesn't work since grouping info is calculated in the function. we could mock it, but I think the other tests cover this.

Reply:
Ah, I see. Okay, great. If we were doing this for real I might suggest moving adding the info into the grouping info helper itself (then you only have to do it in one place, I think?), but again, since this is temporary, I say 🚢 .